journal/src/View/Converter/Block/FigureConverter.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Libero\Journal\View\Converter\Block;
+
+use Libero\Journal\Dom\Element;
+use Libero\Journal\View\Converter\ViewConverter;
+use const Libero\Journal\LIBERO;
+
+final class FigureConverter implements ViewConverter
+{
+    private $converter;
+
+    public function __construct(ViewConverter $converter)
+    {
+        $this->converter = $converter;
+    }
+
+    /**
+     * @param Element $object
+     */
+    public function convert($object, array $context = []) : string
+    {
+        return '<p>[Figure]</p>';
```

Comment:
can be out of scope to do more than this, but it may be useful to hide into the HTML a dump of the original $object to explore what data is available when looking at a page